docs/instrumentation/wsgi/wsgi.rst

```diff
@@ -0,0 +1,7 @@
+OpenTelemetry WSGI Instrumentation
+==================================
+
```

Comment:
Some other instrumentation such as `tornado` and `urllib` etc.. are missing, any particular reason?

Reply:
I think I noticed that earlier too, I just pulled the files that were on Core so I assume they hadn’t been updated on Core either.

For that reason I thought as a follow up we can add those docs.